### DISCUSSION TITLE
How to change the color of the `NavigationIndicator`?

### DISCUSSION THREAD

**Discussion by NafissaBHJ** [bdlukaa/fluent_ui#592] [Q&A] [answered]
Does anyone has an idea how ? 
I searched in the NavigationPane properties but couldn't find the right one to change it, I uploaded the system accent color everywhere else the color has changed but not this one.

<img width="35" alt="Capture" src="https://user-images.githubusercontent.com/20068041/197551936-8adf8486-7402-496d-9e71-5d2b621f5e63.PNG">

**Comment by bdlukaa** [comment, marked as answer]
It uses the `FluentTheme` accent color. You can provide a `ThemeData` in the `FluentApp` and change its `accentColor` property. You can also provide a `StickyNavigationIndicator` to `NavigationPane` with a custom color
